### PR TITLE
Pimp coat remaster

### PR DESCRIPTION
**Pull request by McRamon**
## What Does This PR Do
Resprites pimp coat and adds 2 more:
White one, 2 times cheaper and available in clothes vendor, contraband section.
Tan one - unique coat, that can be found in QM locker, with following stats:
image
Can hold same items as otehr QM suits.

## Why It's Good For The Game
Better lookign drip, more of it and a unique item for QM to feel special. Out of all head jobs QM is the one that feels like could have not exactly regulated attire by corporate standards.

If design team decides that QM shouldn't have this additional unique piece of clothes or balance team somehow thinks that armor is bad balance wise, it can be changed.

## Images of changes
image
image

## Testing
Dressed up some humanised monkeys.

## Changelog
:cl:
add: 2 new pimp coats - one in vendor and 1 specially for QM.
imageadd: New sprites for existing pink coat.
/:cl:

### CODE REVIEW

File: code/modules/clothing/suits/misc_suits.dm [hunk 1]
```diff
@@ -430,6 +430,26 @@
...
 
+	sprite_sheets = list(
+		"Grey" = 'icons/mob/clothing/species/grey/suit.dmi',
+		"Drask" = 'icons/mob/clothing/species/drask/suit.dmi',
+		"Vox" = 'icons/mob/clothing/species/vox/suit.dmi'
+	)
+
+/obj/item/clothing/suit/pimpcoat/white
+	name = "fashionable coat"
+	desc = "Very fluffy white coat, made out of quality synthetic fur."
```
Comment (PopeDaveThe3th) [inline code/modules/clothing/suits/misc_suits.dm:441]:
```suggestion
	desc = "A very fluffy white coat, made out of high-quality synthetic fur."
```

File: code/modules/clothing/suits/misc_suits.dm [hunk 2]
```diff
@@ -430,6 +430,26 @@
...
+	icon_state = "pimpcoat_white"
+	item_state = "pimpcoat_white"
+
+/obj/item/clothing/suit/pimpcoat/tan
+	name = "coat of status"
+	desc = "Very fluffy tan coat, made out of the finest fur from the Earth. Gifted to Quartermaster of Nanotrasen. Has additional layer of protection against harsh environment of Lavaland and a strap to hold equipment of Quartermaster."
```
Comment (PopeDaveThe3th) [inline code/modules/clothing/suits/misc_suits.dm:447]:
```suggestion
	desc = "A very fluffy tan coat, made out of real fur imported from Earth. A symbol of status often given to Nanotrasen Quartermasters. Has an additional layer of protective materials for defense against harsh environments, and a strap to hold equipment."
```

**Review by PopeDaveThe3th — CHANGES_REQUESTED** [review]
Some minor grammar and sentence flow issues but the coats look very nice.

**Review by PopeDaveThe3th — APPROVED** [review]

**Review by Fordoxia — APPROVED** [review]
Proper pimpin'. Taken through the hood, approved, and then taken out to dinner.

**Review by Sirryan2002 — APPROVED** [review]

**Pull request closed without merging**